openassessment/templates/openassessmentblock/staff_debug/student_info.html

```diff
@@ -64,8 +64,35 @@ <h4 class="title--sub">{% trans "Overall Feedback" %}</h4>
                 {% endwith %}
             {%  endfor %}
         </div>
-            {% endif %}
-
+        {% endif %}
```

Comment:
If we restore this indent to its previous level (one more indent in?), and add a newline afterward, we'll remove the "deleted" change and have a cleaner, purely additive diff here.